code/controllers/subsystem/vote.dm

```diff
@@ -205,6 +205,8 @@ SUBSYSTEM_DEF(vote)
 					to_chat(usr, "<span class='warning'>The next ground map has already been selected.</span>")
 					return FALSE
 				var/list/maps = list()
+				if(!config.maplist)
+					return
```

Comment:
All of this is not needed for CBT, but why not

Reply:
Then why did you not atomize it